.github/workflows/ci.yml

```diff
@@ -60,6 +54,10 @@ jobs:
             signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
       - name: Build
         run: nix-build
+      - name: Install
+        run: nix-env -i ./result
+      - name: Test self-upgrade (TODO)
+        run: echo "lorri self-upgrade local $(pwd)"
```

Comment:
`lorri self-upgrade local $(pwd)` blows up right now, I'll deal with that in a separate PR.

Reply:
See #432.